docs/release-notes/8.3.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[release-notes-header-8.3.0]]
+== 8.3
+
+[discrete]
+[[release-notes-8.3.0]]
+== 8.3.0
+
+[discrete]
+[[breaking-changes-8.3.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+* Adds confirmation modal with docs link when updating rules if V1/V2 jobs are installed ({pull}128334[#128334]).
```

Comment:
I'm still working on a good description of this feature and the upgrade/breaking considerations, which I may need to sync with you separately from this PR. For now, a few small tweaks:
```suggestion
* Adds confirmation modal with docs link when updating prebuilt {ml} rules if V1/V2 {ml} jobs are installed ({pull}128334[#128334]).
```